### PR TITLE
follow up PR

### PR DESCRIPTION
**Pull request by SiniWilson**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### CODE REVIEW

File: jazz-build-module/sonar-module.groovy [hunk 1]
```diff
@@ -2,9 +2,8 @@
 import groovy.json.JsonSlurperClassic
 import groovy.json.JsonOutput
 import groovy.transform.Field
-import java.util.regex.*
 
-echo "Sonar code analyzer module loaded successfully"
+	echo "Sonar code analyzer module loaded successfully"
```
Comment (suryajak) [inline jazz-build-module/sonar-module.groovy]: remove indent

File: jazz-build-module/sonar-module.groovy [hunk 2]
```diff
@@ -21,7 +20,7 @@ echo "Sonar code analyzer module loaded successfully"
 @Field def g_sonar_password = ""
 @Field def g_sonar_project_properties
 @Field def g_dependencyCheckOutputFileName = "dependency-check-report.xml"
-@Field def g_dependency_check_properties = [:]
+@Field def g_dependency_check_properties = [: ]
```
Comment (suryajak) [inline jazz-build-module/sonar-module.groovy]: Remove the extra space

File: delete-serverless-service-build-pack/Jenkinsfile [hunk 3]
```diff
@@ -26,7 +26,7 @@ import groovy.transform.Field
 @Field def g_svc_admin_cred_ID = 'SVC_ADMIN'
 @Field def auth_token = ''
 @Field def service_config
-@Field def context_map = [:]
+@Field def context_map = [: ]
```
Comment (suryajak) [inline delete-serverless-service-build-pack/Jenkinsfile]: remove the space here

Comment (SiniWilson) [reply delete-serverless-service-build-pack/Jenkinsfile]: done

File: jenkins-build-pack-lambda/Jenkinsfile [hunk 4]
```diff
@@ -363,7 +363,7 @@ def buildLambda(String runtime) {
 		sh """
 		pip install virtualenv
 		virtualenv venv
-		. venv/bin/activate
+			.venv / bin / activate
```
[the remark this answers is not in the document]
Comment (SiniWilson) [reply jenkins-build-pack-lambda/Jenkinsfile]: done

**Pull request merged**